src/scmdata/dataframe.py

```diff
@@ -536,8 +536,8 @@ def filter(
             - 'level': the maximum "depth" of IAM variables (number of hierarchy levels,
               excluding the strings given in the 'variable' argument)
 
-            - 'time': takes a :class:`datetime.datetime` or list of
-              :class:`datetime.datetime`'s
+            - 'time': takes a :class:`dt.datetime` or list of
```

Comment:
Are you meant to use the local imported name of a module/object in the docs or a fully resolved name?

Reply:
oops didn't mean to change that